dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java

```diff
@@ -13,266 +13,287 @@
-	    attributes.put(SemanticAttributes.THREAD_ID, logData.getThreadId());
-	    
-	    // Add Throwable information to Attribute Builder
-	    String exceptionName = logData.getExceptionName();
-	    String throwable = logData.getThrowable();
-	    if (throwable != null) {
-	    	    attributes.put(SemanticAttributes.EXCEPTION_MESSAGE, exceptionName);
-	    	    attributes.put(SemanticAttributes.EXCEPTION_STACKTRACE, throwable);
-	    }
-	    
-	    // Add additional log information from LogData to Attributes Builder
-	    attributes.put(LogTraceData.getModuleKey(0, isMessageEvent), logData.getModule())
-	              .put(LogTraceData.getMessageIdKey(0, isMessageEvent), logData.getMessageId())
-	              .put(LogTraceData.getMethodNameKey(0, isMessageEvent), logData.getMethodName())
-	              .put(LogTraceData.getClassNameKey(0, isMessageEvent), logData.getClassName())
-	              .put(LogTraceData.getSequenceKey(0, isMessageEvent), logData.getSequence());
-	    
-	    // Set the Attributes to the builder.
-	    builder.setAllAttributes(attributes.build());
-	    
-	    // span context
-	    //builder.setContext(Context.current());
-	}
-	
-	private static Severity wsLevelToSeverity(String level) {
-	    if (level.equals(WsLevel.FATAL.toString())) {
-              return Severity.FATAL;
-	    }
-	    else if (level.equals(WsLevel.SEVERE.toString()) || level.equals("SystemErr")) {
-              return Severity.ERROR;
-            }
-	    else if (level.equals(WsLevel.WARNING.toString())) {
-              return Severity.WARN;
-            }
-	    else if (level.equals(WsLevel.AUDIT.toString())) {
-              return Severity.INFO2;
-            }
-	    else if (level.equals(WsLevel.INFO.toString())) {
-              return Severity.INFO;
-            }
-	    else if (level.equals(WsLevel.CONFIG.toString()) || level.equals("SystemOut")) {
-              return Severity.DEBUG4;
-            }
-            else if (level.equals(WsLevel.DETAIL.toString())) {
-              return Severity.DEBUG3;
-            }
-            else if (level.equals(WsLevel.FINE.toString())) {
-              return Severity.DEBUG2;
+
+@Component(name = OpenTelemetryLogHandler.COMPONENT_NAME, service = { Handler.class }, configurationPolicy = ConfigurationPolicy.OPTIONAL, property = { "service.vendor=IBM" })
+public class OpenTelemetryLogHandler implements SynchronousHandler {
+
+    private static final TraceComponent tc = Tr.register(OpenTelemetryLogHandler.class, "TELEMETRY", "io.openliberty.microprofile.telemetry.internal.common.resources.MPTelemetry");
+
+    public static final String COMPONENT_NAME = "io.openliberty.microprofile.telemetry20.logging.internal.OpenTelemetryLogHandler";
+
+    protected static volatile boolean isInit = false;
+
+    protected CollectorManager collectorMgr = null;
+
+    // Config Source attribute name
+    private static final String SOURCE_LIST_KEY = "source";
+
+    private OpenTelemetryInfo openTelemetry;
```

Comment:
The variable name is confusing.  It is an instance of `OpenTelemetryInfo` rather than `OpenTelemetry`.